biogeochem/EDPhysiologyMod.F90

```diff
@@ -910,9 +910,10 @@ subroutine phenology( currentSite, bc_in )
     ! and thus %nchilldays will never go from zero to 1.  The following logic
     ! when coupled with this fact will essentially prevent cold-deciduous
     ! plants from re-emerging in areas without at least some cold days
-
+    
     if( (currentSite%cstatus == phen_cstat_notcold)  .and. &
-         (dayssincecleafoff > 400)) then           ! remove leaves after a whole year
+         (dayssincecleafoff > 40000)) then           ! remove leaves after a whole year, 
+                                                     !Junyan change dayssincecleafoff > 400 to 40000 to disable this function, so the condition will never be met
```

Comment:
@JunyanDing this looks like a way of preventing "never cold" status.  Is this something particular to your simulation needs, or is there something flawed with how we do "never cold"?

Reply:
@JunyanDing and I are chatting right now.  We both agree that these phenology changes should be reverted out of this PR.

Reply:
Done